examples/cancel_batch.py

```diff
@@ -0,0 +1,107 @@
+# Script that takes in an array of batch names (split by comma) and
+#  applies a bulk action to cancel all tasks in each batch.
+# By default, this script makes 50 concurrent API calls.
+
+# Example: python cancel_batch.py --api_key "SCALE_API_KEY" --batches "batch1,batch2" --clear "True"
+
+import argparse
+from collections import defaultdict
+from concurrent.futures import ThreadPoolExecutor, as_completed
+import sys
+
+import scaleapi
+from scaleapi.exceptions import ScaleException, ScaleUnauthorized
```

Comment:
I think our code linter wants your imports at the very top of the file and then your script explanation below